### ISSUE TITLE
ULS UNITS - PSL - Fix core books accessibility

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#425]
The Core Books section of PSL is currently failing a key accessibility criteria. Some of these books have the exact same name but link to different places. This needs to be fixed as it's a Level A error for Accessibility.

This is something we may be able to fix programmatically by removing the link on the book title and only have it on a call number or something similar that will always be different. If we can't fix it on our end, I would recommend sending this to the PSL stakeholders and having them manually fix these issues.

**Comment by manolobevia** [comment]
Do we have a lot of books with this issue? If not, I would recommend to just update them editing the post.

**Comment by kevinkidwell** [comment]
It looks like there may only be a few because of the way they use spacing, colors, and slashes with the titles (which is terrible but that's beside the point). I think it might still be best if we just remove the hyperlink from the book title and do it on the BIBID field, that would allow each of them to be unique if they are different books with the same titles.
